fv3config/fv3run/_kubernetes.py

```diff
@@ -61,12 +63,18 @@ def run_kubernetes(
             Defaults to "IfNotPresent".
         job_labels (Mapping[str, str], optional): labels provided as key-value pairs
             to apply to job pod.  Useful for grouping jobs together in status checks.
+        **kwargs:  other arguments passed to be passed to run_native. These must be serializeable with json.
     """
+
+    if filesystem._is_local_path(outdir):
+        warnings.warn(
+            f"Output directory {outdir} is a local path, so it will not be accessible "
+            "once the job finishes."
+        )
```

Comment:
This warning might not be possible with a command-centric approach, though we could put it instead in the function that generates the command? Or maybe we'll keep this API the same and it will just make use of the new interface. idk.

Reply:
OK, this was mostly a note about an issue you'll run into in the future - you'll need a special command for submitting kubernetes fv3gfs runs basically so that you can post this warning.